packages/api/src/core.ts

```diff
@@ -26,8 +76,6 @@ function transformCallback<T = unknown>(
 
 class Channel<T = unknown> {
   id: number
-  // @ts-expect-error field used by the IPC serializer
-  private readonly __TAURI_CHANNEL_MARKER__ = true
```

Comment:
@lucasfernog isn't this a breaking change? removing the field will make new `@tauri-apps/api` packages not work with `tauri@2.0.0`